src/MaterialDesignThemes.Wpf/Converters/BooleanToVisibilityConverter.cs

```diff
@@ -2,6 +2,9 @@
 
 public sealed class BooleanToVisibilityConverter : BooleanConverter<Visibility>
 {
+    public static readonly BooleanToVisibilityConverter Instance = new();
+    public static readonly BooleanToVisibilityConverter InverseInstance = new() { FalseValue = Visibility.Visible, TrueValue = Visibility.Hidden };
```

Comment:
I think we probably want a different name here since it is not actually the inverse of `Instance`. Thoughts on `NotHiddenInstance`?

Reply:
Good point. I based it on the "InverseBooleanToVisibilityConverter" that was previously defined as a static resource.
The inverse in this case is based on the inverting of the boolean value not the visibility. I am kind of used to it,
The converters I personally use have 4 instances defined: "HiddenInstance", "HiddenInvertInstance", "CollapsedInstance", "CollapsedInvertInstance". This since we also have cases in which we want to collapse things.

Would it be useful to also introduce these?
So then we would have a "HiddenInstance", "NotHiddenInstance", "CollapsedInstance", "NotCollapsedInstance"

Though I personally prefer the Invert one, but I am a bit biased in this case.

Reply:
Yea I like that naming better, I am fine adding these ones in.